sms_939/models/sms_sponsorship.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+from odoo import models, fields
+
+
+class SponsorshipSMS(models.Model):
+    _name = 'sms_sponsorship'
+
+    partner_id = fields.Id(string='partner_id')
```

Comment:
This should be a Many2one field on table `res.partner` string should be 'Partner' (it's used for the user interface display and should be a friendly name).

Reply:
Done.